src/Console/Commands/CashierUpdate.php

```diff
@@ -36,7 +42,38 @@ public function handle()
 
         $this->comment('Publishing Cashier v2 migrations...');
         $this->callSilent('vendor:publish', ['--tag' => 'cashier-update']);
+        sleep(2);
```

Comment:
What's this for? 😄

Reply:
I try without sleep, and the migration runs until the file published in the migrations folder... Not every time... but is for safety. I can remove it, but local tests don't run migration every time. I try it locally

Reply:
Ok, just keep it